install-and-configure/install/cloud-integration/aws-cloud-integrations/aws-out-of-cluster.md

```diff
@@ -1,72 +0,0 @@
-AWS Out of Cluster 
-==================
-
-Integrating Kubecost with your AWS data provides the ability to allocate out-of-cluster (OOC) costs, e.g. RDS instances and S3 buckets, back to Kubernetes concepts like namespace and deployment as well as reconcile cluster assets back to your billing data. The latter is especially helpful when teams are using Reserved Instances, Savings Plans, or Enterprise Discounts. All billing data remains on your cluster when using this functionality and is not shared externally. Read our [Cloud Integrations](/install-and-configure/install/cloud-integration/README.md) doc for more information on how Kubecost connects with Cloud Service Providers.
-
-The following guide provides the steps required for enabling OOC costs allocation and accurate pricing, e.g. reserved instance price allocation. In a multi-account organization, all of the following steps will need to be completed in the payer account.
-
-## Step 1: Create an AWS Cost and Usage Report (CUR) and integrate it with Kubecost
-
-You can learn how to perform this using our [AWS Cloud Integration](/install-and-configure/install/cloud-integration/aws-cloud-integrations/aws-cloud-integrations.md) doc.
-
-## Step 2: Tag your resources
-Kubecost utilizes AWS tagging to allocate the costs of AWS resources outside of the Kubernetes cluster to specific Kubernetes concepts, such as namespaces, pods, etc. These costs are then shown in a unified dashboard within the Kubecost interface.
-
-To allocate external AWS resources to a Kubernetes concept, use the following tag naming scheme:
-
-| Kubernetes Concept| AWS Tag Key | AWS Tag Value |
-|--------------------|---------------------|---------------|
-| Cluster | kubernetes_cluster	| cluster-name	|
-| Namespace | kubernetes_namespace	| namespace-name |
-| Deployment | kubernetes_deployment	| deployment-name |
-| Label | kubernetes\_label\_NAME* | label-value    |
-| DaemonSet | kubernetes_daemonset	| daemonset-name |
-| Pod | kubernetes_pod	    | pod-name     |
-| Container | kubernetes_container	| container-name |
-
-{% hint style="info" %}
-In the `kubernetes_label_NAME` tag key, the `NAME` portion should appear exactly as the tag appears inside of Kubernetes. For example, for the tag `app.kubernetes.io/name`, this tag key would appear as `kubernetes_label_app.kubernetes.io/name`.
```

Comment:
I think this detail is still important/relevant?

Reply:
@thomasvn do you know where this info is relevant? My understanding is that these tagging instructions were only really relevant for 'external costs' (the removed column in allocations page)

Reply:
@kwombach12 Ah that's right. I was thinking back to Advanced Reports as well. Do Collections need this level of tagging?

Reply:
@thomasvn no -- collections leverages the same filtering as Cloud Costs so this tagging doesn't really enhance any existing features of our product as far as I can tell. I've also seen a couple cases where users get confused by these docs -- they do all this tagging and expect kubecost to allocate/handle that mapping and we don't anymore

Reply:
Right, makes sense. Thanks for clarifying! I agree we should remove this section of the docs then